tests/resources/Engine.py

```diff
@@ -647,6 +714,9 @@ def build_configs(self, hosts: int, services_by_host: int, debug_level=0):
             with open(f"{config_dir}/escalations.cfg", "w") as f:
                 pass
 
+            f = open(config_dir + "/meta_services.cfg", "w")
+            f.close()
+
```

Comment:
Do we really need to create meta_services.cfg?

Reply:
no so i removed it